crit/Makefile

```diff
@@ -1,5 +1,12 @@
 
-all-y	+= crit
+all-y	+= crit decoder
+
+CC	= gcc
+CFLAGS  = -g -Wall
+
+decoder:
+	$(Q) $(CC) $(DEFINES) crit/decoder.c -o crit/decoder
```

Comment:
Please, add it to .gitignore so that git gets to know that it doesn't need track changes in the binary.
```
diff --git a/.gitignore b/.gitignore
index 23cd703be2f5..83368327de5d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -25,6 +25,7 @@ images/google/protobuf/*.h
 .gitid
 criu/criu
 crit/crit
+crit/decoder
 criu/arch/*/sys-exec-tbl*.c
 # x86 syscalls-table is not generated
 !criu/arch/x86/sys-exec-tbl.c
```